CMakeLists.txt

```diff
@@ -68,12 +68,23 @@ endif()
 if("${MD_SIM}" STREQUAL "SIMPLE_MD" AND BUILD_WITH_LAMMPS)
   message(WARNING ">> BUILD_WITH_LAMMPS is enabled but MD_SIM is SIMPLE_MD. Are you sure?")
 endif()
+
 # Compiler list to be used for compiletest
 set(COMPILETEST_COMPILERS "g++ clang++" CACHE STRING "Compiler list to be used for compiletest")
 message(STATUS ">> Compiler list to be used for compiletest : " ${COMPILETEST_COMPILERS})
-# Enable coupling with the preCICE API, default is off
+
+# Build mode list to be used for compiletest
+set(COMPILETEST_MODES "Debug DebugOptimized RelWithDebInfo Release" CACHE STRING "Build mode list to be used for compiletest")
+message(STATUS ">> Build mode list to be used for compiletest : " ${COMPILETEST_MODES})
+
+# Enable coupling with indexing tests, default is off
 option(BUILD_WITH_TEST_INDEXING "Enable indexing tests, default is off." OFF)
 message(STATUS ">> Build with indexing tests : " ${BUILD_WITH_TEST_INDEXING})
+
+# Enable coupling with pint debug, default is off
+option(BUILD_WITH_PINT_DEBUG "Enable parallel-in-time integration debugging, default is off." OFF)
```

Comment:
Yes I agree that it looks like a typo of “print”, indeed, but no, I do not think we should try to use different wording. There are many words and acronyms that look similar but have a very different meaning. Once people get used to them and get trained to recognize the difference, it is not a problem any more. In this case, we want the reader to learn the acronym (because it is already common in the community) and its meaning is given here.